src/components/_basic.less

```diff
@@ -44,6 +44,16 @@ a.sbtn {
   border-radius: 25px;
 }
 
+button:disabled {
```

Comment:
Please use `.sbtn:disabled` instead of the general `button:disabled`

Reply:
done! please review @shahednasser ! thank you!